src/style.css

```
@@ -1,3 +1,203 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+/* body {
+  line-height: 1;
+} */
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+html {
+  --link-hover-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  background-color: #ededed;
+
+  line-height: 1;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.header__link,
+.nav__link {
+  text-decoration: none;
+  line-height: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+nav > ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  font-size: 12px;
+  font-weight: 500;
+
+  height: 60px;
+  line-height: 60px;
+
+  text-align: center;
+  color: #000;
+  text-transform: uppercase;
+  margin-left: 20px;
+}
+
+.nav__link:hover,
+.is-active {
+  color: var(--link-hover-color);
+}
+
+nav li:first-child > .nav__link {
+  margin-left: 0;
+}
```

```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```
looks like we can simplify it